cvat/schema.yml

```diff
@@ -11039,6 +11058,14 @@ components:
           allOf:
           - $ref: '#/components/schemas/StorageRequest'
           nullable: true
+        consensus_replicas:
+          type: integer
+          minimum: 0
+          nullable: true
```

Comment:
Why should we allow `null` here? Presumably 0 means "consensus disabled", so we don't need another way to specify that.

Reply:
Probably, it was supposed to mean to use the default value, because it's a more convenient way to do this than using kwargs in SDK. Ok, let's remove it.